lib/order.rb

```diff
@@ -0,0 +1,86 @@
+require_relative "customer.rb"
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id #number
+    @customer = customer #object
+    @products = products #hash, { "banana" => 1.99, "cracker" => 3.00 }
+    @fulfillment_status = fulfillment_status
+    required_fulfillment_status = [:pending, :paid, :processing, :shipped, :complete]
+    if required_fulfillment_status.include?(@fulfillment_status)
+    else
+      raise ArgumentError
```

Comment:
Two things:
1. This list of statuses would be a great thing to store in a constant.
1. When you see an empty `if` clause followed by an `else`, that's a good indicator that you should use `unless` instead.